Update virtual-machines-common-sizes-memory.md

larryclaman opened the pull request:
update virtual-machines-common-sizes-memory.md to fix incorrect performance numbers for standard_E3 types.  Used perf numbers from blog post @ https://azure.microsoft.com/en-us/blog/introducing-the-new-dv3-and-ev3-vm-sizes/

tysonn commented:
I asked Tim or Kathy to take a look at the numbering issue - publishing won't fix it, so it needs to be fixed in the markdown.

tysonn commented:
Kathy incorporated these changes, so closing the PR

larryms committed:
Update virtual-machines-common-sizes-memory.md

update virtual-machines-common-sizes-memory.md to fix incorrect performance numbers for standard_E3 types.  Used perf numbers from blog post @ https://azure.microsoft.com/en-us/blog/introducing-the-new-dv3-and-ev3-vm-sizes/
